test/proxy.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * The talkback proxy for e2e tests. When making a request to the API during testing,
+ * it tries to use the responses it previously saved in `/test/tapes` folder. If no
+ * response is found there, it makes an actual request, and saves the response for
+ * future use.
+ * This makes it possible for the e2e tests to run without internet, and makes the
+ * tests less flaky due to changes in the API or API data.
+ */
+const talkback = require('talkback')
+const host = 'https://api.openverse.engineering/v1'
+
+const tapeNameGenerator = (tapeNumber) => `response-${tapeNumber}`
+
+const opts = {
+  host,
+  port: 3000,
+  path: './test/tapes',
+  record: talkback.Options.RecordMode.NEW,
+  debug: false,
+  name: 'Openverse e2e proxy',
+  tapeNameGenerator,
+}
```

Comment:
Should we add the `ignoreHeaders` option to ignore `user-agent`, `origin` and `referrer`? For each one, here's why I think we shouldn't match tapes against them:

* `user-agent`: This differs between platforms so even just running in CI or updating browser versions could cause a mismatch (though Chrome has frozen their UA string I think; but the tapes wouldn't match if we started testing multiple UAs like Firefox, Safari or mobile browsers)
* `origin`: This will differ between environments, in particular the plan for the visual regression testing RFC, we'll run those e2e tests inside a docker container (we could run _all_ playwright tests in docker if we wanted) and in that case the origin would either be `host.docker.internal` to allow for accessing the host's localhost; or if we use docker networking and run the frontend in docker as well as the e2e tests then it'd be the frontend container's hostname.
* `referrer`: Same as `origin`.

We'd also want to include the two defaults of `content-length` and `host` I guess.